lightning/src/chain/channelmonitor.rs

```diff
@@ -1210,6 +1210,10 @@ impl<ChanSigner: ChannelKeys> ChannelMonitor<ChanSigner> {
 			feerate_per_kw: initial_local_commitment_tx.feerate_per_kw,
 			htlc_outputs: Vec::new(), // There are never any HTLCs in the initial commitment transactions
 		};
+
+		let mut outputs_to_watch = HashMap::new();
```

Comment:
Seems like a regression test should go with the fix. How does the test infrastructure look around this code?

Reply:
IIUC, `outputs_to_watch` was added but not used in such a way that required the funding txo. When I began using it in such a manner, a few tests broke. This commit is what I used to fix the tests, though I placed the commit prior to the breaking change.

Reply:
I see. I just didn't see the cause and effect in my pass through. Thanks for the background.